dspace-api/src/main/java/org/dspace/administer/RegistryLoader.java

```diff
@@ -47,7 +47,7 @@
     /**
      * log4j category
      */
-    private static Logger log = Logger.getLogger(RegistryLoader.class);
+    private static Logger log = org.apache.logging.log4j.LogManager.getLogger(RegistryLoader.class);
```

Comment:
I am not sure how to simply this with causing a conflict with `import org.dspace.core.LogManager`

Reply:
That is the only way that I know.  Any class which uses org.dspace.core.LogManager *and* org.apache.logging.log4j.LogManager will be able to import at most one, and must give the fully qualified name of the other wherever it is used.  Since there should be at most one reference to org.apache.logging.log4j.LogManager in any class, that is the one to spell out.